src/views/modals/LabelsModal/LabelsModal.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
```

Comment:
Was this copied over from console? We should avoid reimplementing console functionality and instead open an issue to expose the labels modal.